internal/providers/terraform/google/logging_organization_sink_test.go

```diff
@@ -0,0 +1,112 @@
+package google_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/testutil"
+	"github.com/shopspring/decimal"
+)
+
+func TestLoggingOrgSink(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	tf := `
```

Comment:
Do the tests work ok if you just have this:
```
resource "google_logging_organization_sink" "basic" {
  name   = "basic"
	description = "what it is"
	org_id = "123456789"
	destination = "storage.googleapis.com/fake"
}
```

If so, please update them to simplify them a bit (the same approach should work for the other test files).